src/daft-core/src/kernels/hashing.rs

```diff
@@ -93,6 +95,26 @@ fn hash_binary<O: Offset>(
     PrimitiveArray::<u64>::new(DataType::UInt64, hashes.into(), None)
 }
 
+fn hash_fixed_size_binary(
+    array: &FixedSizeBinaryArray,
+    seed: Option<&PrimitiveArray<u64>>,
+) -> PrimitiveArray<u64> {
+    let hashes = if let Some(seed) = seed {
+        array
+            .values_iter()
+            .zip(seed.values_iter())
+            .map(|(v, s)| xxh3_64_with_seed(v, *s))
+            .collect::<Vec<_>>()
+    } else {
+        println!(
+            "array.values_iter(): {:?}",
+            array.values_iter().collect::<Vec<_>>()
+        );
```

Comment:
@colin-ho should this print be removed?
```suggestion
```

Reply:
Ah yes, I sped run this PR so there may be some cleaning up to do 😭